explorer/app/components/common/Paper/paper.tsx

```diff
@@ -3,22 +3,21 @@
  */
 
 import { Paper as Panel, PaperProps } from "@mui/material";
-import React, { ReactNode } from "react";
+import React, { forwardRef, ReactNode } from "react";
```

Comment:
ForwardRef is required as we use a styled component (RoundedLoading) to display loading component and we need to pass the loading ref on to MuiPaper.